src/common_types.ts

```diff
@@ -255,6 +255,7 @@ export type PushOptions = Partial<ProjectSettings> &
     kibanaVersion?: string;
     yes?: boolean;
     tags?: Array<string>;
+    labels?: Record<string, string>;
```

Comment:
Let's rename this to fields to map with the heartbeat top level fields name and under the hood in Kibana, you can map to labels. Having labels as top level is confusing - 

Also let's write JS doc annotation so it's visible on TS config.